package.json

```diff
@@ -7,15 +7,15 @@
     "test": "test"
   },
   "scripts": {
-    "test": "npm run lint && npm run coverage",
-    "coverage": "NODE_PATH=./lib NODE_ENV=test istanbul cover _mocha -- --recursive test/unit test/functional/ && npm run coverage-check",
-    "coverage-unit": "NODE_PATH=./lib NODE_ENV=test istanbul cover _mocha --dir coverage-unit -- --recursive test/unit",
-    "coverage-functional": "NODE_PATH=./lib NODE_ENV=test istanbul cover _mocha --dir coverage-unit -- --recursive test/functional",
+    "test": "GITHUB_USERNAME=runnabear npm run lint && npm run coverage",
```

Comment:
Maybe just put `runnabear` in the `config/.env` file. Not sure why I had that be empty...